nixos/doc/manual/release-notes/rl-1909.xml

```diff
@@ -284,6 +284,16 @@
        Squid 3 has been removed and the <option>squid</option> derivation now refers to Squid 4.
      </para>
    </listitem>
+   <listitem>
+     <para>
+       The <option>security.acme.directory</option> option has been replaced by a read-only <literal>directory</literal> option for each certificate you define. The
+       <option>security.acme.preDelay</option> and <option>security.acme.activationDelay</option> options have been removed. To execute a service before certificates
+       are provisioned or renewed add a <literal>RequiredBy=acme-${cert}.service</literal> to any service.
```

Comment:
We should add that this is already done if `services.nginx.virtualHosts.<name>.enableACME` is set to `true`, so people don't wonder if they now need to add this to their nginx configuration.

Reply:
rebased and added to the docs